e2e/specs/st_map.spec.js

```diff
@@ -23,4 +23,8 @@ describe("st.map", () => {
   it("displays 3 maps", () => {
     cy.get(".element-container .stDeckGlJsonChart").should("have.length", 3);
   });
+  
+  it("displays 3 zoom buttons", () => {
+    cy.get(".element-container .zoomButton").should("have.length", 3)
+  })
```

Comment:
Not sure if it's possible, but should we have a screenshot test? This would allow us to ensure the buttons don't move around without intention.

It's possible it might be difficult or flaky, but I figure we would want that.